File: Lesson07/ListViewCustomAdapter2/app/src/main/java/com/codingblocks/listviewcustomadapter/NewActivity.java
package com.codingblocks.listviewcustomadapter;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.LinearLayout;

public class NewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_new);

        LinearLayout parent = findViewById(R.id.linearLayout);

        View inflatedView = getLayoutInflater()
                .inflate(R.layout.item_row,parent,false);

        parent.addView(inflatedView);

    }
}

File: Lesson15/Parcelable/app/src/main/java/com/codingblocks/parcelable/NewActivity.kt
package com.codingblocks.parcelable

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class NewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_new)

        val note = intent.getParcelableExtra<Note>("NOTE")
        val notes = intent.getParcelableArrayListExtra<Note>("NOTES")
//        val noteString = intent.getStringExtra("NOTE")
//        val note = gson.fromJson(noteString, Note::class.java)

    }
}
